packages/frontend/src/components/DashboardTiles/DashboardChartTile.tsx

```diff
@@ -508,6 +508,20 @@ const DashboardChartTile: FC<Props> = (props) => {
                                                         pivotReference,
                                                         dashboardFiltersThatApplyToChart,
                                                     );
+                                                    track({
+                                                        name: EventName.VIEW_UNDERLYING_DATA_CLICKED,
+                                                        properties: {
+                                                            organizationId:
+                                                                user?.data
+                                                                    ?.organizationUuid,
+                                                            userId: user?.data
+                                                                ?.userUuid,
+                                                            projectId:
+                                                                projectUuid,
+                                                            context:
+                                                                'dashboard',
```

Comment:
We are only recording the context in this file. 

I think this context could be tracked via the section metadata. Tracking docs: https://github.com/lightdash/lightdash/blob/main/.github/docs/Tracking.md
Eg:
```<TrackSection name={SectionName.DASHBOARD_TILE}>```

Reply:
yep, I just forgot to remove this last one!
thx